english/eating-walls/find-a-toilet/door/pushed/pushed.md

```diff
@@ -0,0 +1,10 @@
+A tall thin man in jester's motley stands before you. He sighs and lights a cigarette.
+
+You look down and realize you have just wet your britches.  
+
+The man looks at you and riggles his fingers in the air like he's really spooky. "HAH! Skiddish are ya? Well, too late to show you to the restrooms..."

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.